documentation/docs/tutorials/openfga.mdx

```diff
@@ -0,0 +1,105 @@
+# OpenFGA and OPAL
+
+[OpenFGA](https://openfga.dev) is an open-source authorization engine implementing the Zanzibar authorization model created by Google. OPAL provides realtime synchronization and management capabilities for OpenFGA, allowing you to keep your authorization data and relationships up-to-date across distributed OpenFGA instances.
+
+## Quick Start
+
+OPAL can run OpenFGA as a policy engine instead of OPA. To launch an example configuration with Docker-Compose:
```

Comment:
> OPAL can run OpenFGA as a policy engine instead of OPA.

This is redundant. If you'd like to write that OPAL is supports multiple policy engines, it should be in the introduction.